Description: Fix bashisms in /bin/sh scripts
Author: Valentin Vidic <vvidic@debian.org>
Last-Update: 2020-06-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/heartbeat/mdraid
+++ b/heartbeat/mdraid
@@ -557,7 +557,7 @@
 [ "$1" = "validate-all" ] && return ${OCF_SUCCESS}
 
 # Required by start|stop|monitor processed below
-devs=(`list_devices_for_mddev`)
+devs="`list_devices_for_mddev`"
 if [ `echo $devs|wc -l` -eq 0 ]; then
 	ocf_exit_reason "No component device(s) found for MD RAID array $md_dev"
 	exit $OCF_ERR_GENERIC
